tests/__mocks__/results/output-reward-split.html

```diff
@@ -1,4 +1,4 @@
 <!-- Ubiquity - GithubCommentModule - GithubCommentModule.createStructuredMetadata - 1234
```

Comment:
```suggestion
<!-- Ubiquity - GithubCommentModule - GithubCommentModule.createStructuredMetadata - 28d3b8841b6a91baa874b1cf0f1d95a307e296d7
```

This should be the revision hash, is this not implemented correctly?

Reply:
This is a mocked version, otherwise that value would change every commit and would crash the test all the time.